src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -5804,6 +5828,74 @@ public enum EventManifestOptions
         AllowEventSourceOverride = 0x8,
     }
 
+#if FEATURE_EVENTSOURCE_XPLAT 
+
+    internal  class XplatEventLogger : EventListener
```

Comment:
This is actually Linux specific for now, I was under the impression that we are going to follow the same design for all non windows platform or am i wrong ?.